docs/installation.md

```diff
@@ -75,9 +75,6 @@ If you want to help develop Lychee, install the development dependencies by remo
 #### Public Directory
 After installing Lychee, configure your web server's root to be the `public` directory. The `index.php` in this directory serves as the interface for all HTTP requests to Lychee's API. Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
 
-#### Configuration Files
-All of the configuration files for Lychee are stored in the `config` directory. Each option is documented, so feel free to look through the files and get familiar with the options available to you.
-
 #### Directory Permissions
 After installing Lychee, you need to configure some permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or Lychee will not run. Additionaly `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
```

Comment:
The "after installing Lychee" is repeated from some lines above. Again, I can only apologize and give the lame excuse that I have never read the instructions thoroughly. That is what happens when you invite me to a review :)

I would rephrase it like this:

The phrase "after installing..." should only be used once and put directly below the 3rd-level caption `### Configuration`. This also avoids having no text between two consecutive captions. Something like this "After installing Lychee some adjustments must be made as detailed out below". Then remove that phrase from everywhere else, because all of these steps are after the installation.

Reply:
I'd go further and say that configuration is inherently going to be after installation.